app/src/main/java/com/dcrandroid/activities/privacy/SetupPrivacy.kt

```diff
@@ -25,14 +34,82 @@ class SetupPrivacy : BaseActivity() {
         wallet_name.text = wallet.name
 
         btn_setup_mixer.setOnClickListener {
-            val intent = Intent(this, SetupMixerAccounts::class.java)
-            intent.putExtra(Constants.WALLET_ID, wallet.id)
-            finish()
-            startActivity(intent)
+            Toast.makeText(this, wallet.accountsRaw.toString(), Toast.LENGTH_SHORT).show()
+            if (wallet.accountsRaw.count > 1) {
```

Comment:
There's a hidden account created for every wallet called "imported", So checking if the wallet has just one account, wouldn't work as intended.

We could check if a wallet has more that one account and also check if the wallet doesn't have any accounts with the name "imported" before hiding that page